packages/styles/scss/components/masthead/_masthead-search.scss

```diff
@@ -125,6 +125,11 @@
     position: absolute;
     top: 0;
     right: 0;
+
+    .#{$prefix}--header__search--search,
+    .#{$prefix}--header__search--close {
+      color: #000000;
```

Comment:
think we can use the `$ui-05` token instead?

Reply:
```suggestion
      color: $icon-01;
```